modules/sharethroughBidAdapter.js

```diff
@@ -148,4 +149,25 @@ function b64EncodeUnicode(str) {
       }));
 }
 
+function canAutoPlayHTML5Video() {
+  var userAgent = navigator.userAgent;
+  if (!userAgent) return false;
+
+  var isAndroid = /Android/i.test(userAgent);
```

Comment:
A small request - can you update these `var`s to either `let` or `const` so it's consistent with the rest of the file?

Reply:
Updated. Thank you for reviewing!